initial_joint:
  body_lift_joint: 0.05
  body_roll_joint: 0
  head_tilt_joint: 0
  head_pan_joint: 0
  right_shoulder_roll_joint: 0
  right_shoulder_flex_joint: 0.40
  right_wrist_flex_joint: 0
  right_hand_motor_joint: 0
  left_shoulder_roll_joint: 0
  left_shoulder_flex_joint: -0.40
  left_wrist_flex_joint: 0
  left_hand_motor_joint: 0
